mne_qt_browser/_pg_figure.py

```diff
@@ -4234,6 +4232,80 @@ def __init__(self, **kwargs):
             # disable histogram of epoch PTP amplitude
             del self.mne.keyboard_shortcuts["h"]
 
+        self.create_menus()
+
+    def create_menus(self):
+        """Creates and returns the application's menu bar with properly aligned shortcuts."""
+        if not self.menuBar():
+            self.setMenuBar(QMenuBar(self))
+        menu_bar = self.menuBar()
+        mne_python = menu_bar.addMenu("MNE-Python")
+        view_menu = menu_bar.addMenu("View")
+        help_menu = menu_bar.addMenu("Help")
+        scroll_menu = view_menu.addMenu("Scroll")
+
+        def add_menu_action(menu, description, shortcut):
+            widget = QWidget()
+            layout = QHBoxLayout(widget)
+            layout.setContentsMargins(10, 2, 10, 2)
+
+            desc_label = QLabel(description)
+            shortcut_label = QLabel(shortcut)
+
+            layout.addWidget(desc_label)
+            layout.addStretch()
+            layout.addWidget(shortcut_label)
+
+            action = QWidgetAction(self)
+            action.setDefaultWidget(widget)
+            menu.addAction(action)
+
+        shortcut_data = {
+            mne_python: [
+                ("Create annotation     ", "Click + Drag"),
+                ("Mark/unmark bad channel     ", "Left-click channel"),
+                ("Mark/unmark bad segment     ", "Left-click data"),
+                ("Toggle Annotation Tool / Visibility     ", "A"),
+                ("Toggle Butterfly     ", "B"),
```

Comment:
Looks like a good start. A few ideas:

1. From a UI perspective, a clickable menu item should initiate an action. Some of these can't, such as "Create Annotation". These should at least be "disabled" (i.e., still visible but not clickable). Not 100% sure we want these in a menu but... maybe okay to leave them.
2. Some of them when clicked should initiate the action, such as the "Toggle DC Correction" -- clicking it should actually toggle DC correction.

Also this code isn't quite DRY... every first option has a bunch of extra spaces at the end. (1) I don't think these should be needed, and (2) if they absolutely are, they could be added to `add_menu_action` rather than being in every one of these entries.